es_ES/faq.md

```diff
@@ -0,0 +1,625 @@
+propio gestor de reconocimiento que es un participante de primera clase en el
+[gesture system](/gestures/). Por otra parte, dos widgets
+escrito por diferentes personas puede coordinar para desambiguar los gestos.
+
+Las tendencias de diseño de aplicaciones modernas apuntan a diseñadores y usuarios que desean UI más ricas en movimiento y diseños propios de la marca.
+Para alcanzar ese nivel de diseño personalizado y hermoso,
+Flutter está diseñado para manejar píxeles en lugar de los widgets OEM.
+
+Al usar el mismo renderizador, framework y conjunto de widgets, lo hacemos
+más fácil de publicar para iOS y Android al mismo tiempo, sin tener que hacer
+planificación cuidadosa y costosa para alinear dos bases de códigos y conjuntos de características por separado.
+
+Al usar un solo idioma, un solo framework,
+y un único conjunto de bibliotecas para toda su UI
+(independientemente de si su interfaz de usuario es diferente para cada plataforma móvil o en gran medida consistente), también apuntamos a
+ayuda a reducir los costos de desarrollo y mantenimiento de la aplicación.
+
+### Qué sucede cuando mi sistema operativo móvil se actualiza e introduce nuevos widgets?
+
+El equipo de Flutter observa la adopción y demanda de nuevos widgets para dispositivos móviles de iOS y Android, y tiene como objetivo trabajar con la comunidad para construir soporte para nuevos widgets. Este trabajo puede venir en la forma de las características del framework de nivel inferior, nuevos widgets compuestos, o nuevas implementaciones de widgets.
+
+La arquitectura en capas de Flutter está diseñada para soportar numerosas
+bibliotecas de widgets, y alentamos y apoyamos a la comunidad en
+construir y mantener bibliotecas de widgets.
+
+### Qué sucede cuando mi sistema operativo móvil se actualiza e introduce nuevas capacidades de plataforma?
+
+El sistema de interoperabilidad y plugin de Flutter está diseñado para permitir
+desarrolladores para acceder a las nuevas funciones y capacidades del sistema operativo móvil inmediatamente. Los desarrolladores no tienen que esperar al equipo de Flutter para exponer la nueva capacidad del sistema operativo móvil.
+
+### Qué sistemas operativos puedo usar para construir una aplicación de Flutter?
+
+Flutter admite el desarrollo en Linux, Mac y Windows.
+
+### En qué idioma está escrito Flutter?
+
+Observamos muchos lenguajes y tiempos de ejecución, y finalmente adoptamos a Dart para el framwork y los widgets. El framework de gráficos subyacente y la máquina virtual de Dart se implementan en C/C++.
+
+### Por qué Flutter eligió usar Dart?
+
+Flutter utilizó cuatro dimensiones principales para la evaluación, y consideró las necesidades de autores de framework, desarrolladores y usuarios finales. Encontramos algunos lenguajes encontrados algunos requisitos, pero Dart obtuvo una puntuación alta en todas nuestras dimensiones de evaluación y cumplió todos nuestros requisitos y criterios.

```

Comment:
We found some languages met some requirements
Encontramos que algunos lenguajes cumplían algunos de los requisitos, pero Dart....